client/ayon_core/hosts/houdini/plugins/publish/extract_mantra_ifd.py

```diff
@@ -33,9 +33,9 @@ def process(self, instance):
                 os.path.normpath(os.path.join(staging_dir, frame)))
         ]
         if missing_frames:
-            raise RuntimeError("Failed to complete Mantra ifd extraction. "
-                               "Missing output files: {}".format(
-                                   missing_frames))
+            raise KnownPublishError("Failed to complete Mantra ifd extraction. ",
+                                    # "Missing output files: {}".format(missing_frames),
```

Comment:
I guess we'll still need to report this somewhere? E.g. having a clear `log.error()` prior to this?

Reply:
Agreed! 
going to push a commit shortly.

![image](https://github.com/ynput/ayon-core/assets/20871534/d985412f-1918-4302-968b-c70196cde068)

Reply:
here it is https://github.com/ynput/ayon-core/pull/373/commits/e6c52f0430bd63cf8f3a469d9c7edae94d3ad8fd